website/src/components/BlogsSlider/style.ts

```diff
@@ -10,7 +10,19 @@ const useStyles = makeStyles((theme: Theme) => ({
   media: {
     height: 0,
     paddingTop: "56.25%",
-    borderRadius: '24px 24px 24px 0px',
+    borderRadius: "24px 24px 24px 0px",
+    position: 'relative',
+    overflow: 'hidden',
+    '& img':{
+      minWidth: '100%',
```

Comment:
here minWidth and maxWidth value is same, we should use `width: 100%` and remove other two `minWidth` and `maxWidth` property.